authentication/jwtTokenProvider.go

```diff
@@ -10,9 +10,22 @@ var (
     defaultKeyFunc    jwt.Keyfunc = func(t *jwt.Token) (interface{}, error) { return jwtTestDefaultKey, nil }
 )
 
+type MyTimeClaims struct {
+    exp int64
+    jwt.StandardClaims
+}
+
+
 func GenerateToken(signingKey []byte) (string, error) {
-    token := jwt.New(jwt.SigningMethodHS256)
-    token.Claims["exp"] = time.Now().Add(time.Hour * 24).Unix()
+    claims := MyTimeClaims{
+        time.Now().Add(time.Hour * 24).Unix(),
+        jwt.StandardClaims{
+            ExpiresAt: 15000,
```

Comment:
Can we configure this rather than hard code? We need a config system across the app maybe start it here?